data-raw/07_FLDAS_SWE.R

```diff
@@ -0,0 +1,80 @@
+overwrite_csv <- TRUE
+
+box::use(
+  rgee[...],
+  tidyrgee[...],
+  readr[...],
+  janitor[...],
+  dplyr[...],
+  logger,
+  AzureStor,
+  ../R/blob_connect
+
+  )
+# ee_check()
+ee_Initialize()
```

Comment:
ah yeah soo annoying... The only thing i can think of is just to follow the `{rgee}` defaults and have it install an `{rgee}` conda env thats used across projects whenever `{rgee}` is used. That's what I have always done, and while it still has some issues/complexities, it might be the smoothest route